plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixClient.java

```diff
@@ -201,10 +201,10 @@ public PhoenixClient(PhoenixConfig config, ConnectionFactory connectionFactory)
         getConnectionProperties(config).forEach((k, v) -> configuration.set((String) k, (String) v));
```

Comment:
Please improve commit message. You are actually fixing a connection leak due lazy connection factory. While your commit messages sounds like a code cleanup.

Reply:
I didn't know I am fixing a leak. Do you know the mechanics of it?

Reply:
Well... you are right. It is not yet actual leak, it is just inconsistent. The LazyConnection is opened, but not closed. It is not a real leak as a real connection is closed.

Let's keep the commit message then.

Reply:
Thanks for confirming!